### PR TITLE
Distribute embark-org, Linting

### PR DESCRIPTION
**Pull request by minad**

### CODE REVIEW

**Comment by oantolin** [comment]
Thanks for the strong hint it was time to publish embark-org. I guess I need to do a PR on MELPA, too.

File: .elpaignore [hunk 1]
```diff
@@ -1 +1 @@
-embark-org.el
\ No newline at end of file
+LICENSE
```
Comment (oantolin) [inline .elpaignore:1]: Wait, is ignoring the LICENSE correct?

Comment (oantolin) [reply .elpaignore:1]: Or am I going to get a distressed email from Stefan about removing the LICENSE file? :)

Comment (minad) [reply .elpaignore:1]: Yes, it is fine. Stefan added many LICENSE ignores in elpa-packages himself.

**Comment by minad** [comment]
> I guess I need to do a PR on MELPA, too.

Indeed, MELPA only distributes embark.el. 

https://github.com/melpa/melpa/blob/master/recipes/embark#L3

File: .elpaignore (hunk 1 above)
Comment (oantolin) [reply .elpaignore:1]: OK, good. I thought you probably knew for a fact this was OK, but wanted to confirm just in case.

**Comment by oantolin** [comment]
I made a PR on MELPA: melpa/melpa#8294

**Pull request merged**